test/CMakeLists.txt

```diff
@@ -94,9 +94,8 @@ list(APPEND TEST_LIST
     test_34_connect_locally_with_remote_server
 )
 
-# Adding remote servers to the server using env var was introduced in Fast DDS v2.3. Only checking
-# minor version because find_package already checks major.
-if(fastrtps_VERSION_MINOR GREATER_EQUAL 3)
+# Adding remote servers to the server using env var was introduced in Fast DDS v2.3.
+if(fastrtps_VERSION VERSION_GREATER_EQUAL "2.3.0")
```

Comment:
Is there going to be a backport of this feature to \<2.2.x\>?

Reply:
No, this will be a feature of 2.3